{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2024-3772",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "Regular expression denial of service in Pydanic < 2.4.0, < 1.10.13 allows remote attackers to cause denial of service via a crafted email string.\n",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2024-3772",
				"url": "https://www.suse.com/security/cve/CVE-2024-3772"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1222806 for CVE-2024-3772",
				"url": "https://bugzilla.suse.com/1222806"
			}
		],
		"title": "SUSE CVE CVE-2024-3772",
		"tracking": {
			"current_release_date": "2024-08-03T02:55:09Z",
			"generator": {
				"date": "2024-04-15T23:13:24Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2024-3772",
			"initial_release_date": "2024-04-15T23:13:24Z",
			"revision_history": [
				{
					"date": "2024-04-15T23:13:24Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2024-04-17T02:22:04Z",
					"number": "3",
					"summary": "Current version"
				},
				{
					"date": "2024-07-03T03:40:48Z",
					"number": "4",
					"summary": "Current version"
				},
				{
					"date": "2024-08-03T02:55:09Z",
					"number": "5",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "5"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-pydantic-2.6.4-2.1",
								"product": {
									"name": "python310-pydantic-2.6.4-2.1",
									"product_id": "python310-pydantic-2.6.4-2.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-pydantic-2.6.4-2.1",
								"product": {
									"name": "python311-pydantic-2.6.4-2.1",
									"product_id": "python311-pydantic-2.6.4-2.1"
								}
							},
							{
								"category": "product_version",
								"name": "python312-pydantic-2.6.4-2.1",
								"product": {
									"name": "python312-pydantic-2.6.4-2.1",
									"product_id": "python312-pydantic-2.6.4-2.1"
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-pydantic-2.6.4-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-pydantic-2.6.4-2.1"
				},
				"product_reference": "python310-pydantic-2.6.4-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-pydantic-2.6.4-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-pydantic-2.6.4-2.1"
				},
				"product_reference": "python311-pydantic-2.6.4-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-pydantic-2.6.4-2.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-pydantic-2.6.4-2.1"
				},
				"product_reference": "python312-pydantic-2.6.4-2.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2024-3772",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-3772"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Regular expression denial of service in Pydanic < 2.4.0, < 1.10.13 allows remote attackers to cause denial of service via a crafted email string.\n",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-pydantic-2.6.4-2.1",
					"openSUSE Tumbleweed:python311-pydantic-2.6.4-2.1",
					"openSUSE Tumbleweed:python312-pydantic-2.6.4-2.1"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-3772",
					"url": "https://www.suse.com/security/cve/CVE-2024-3772"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1222806 for CVE-2024-3772",
					"url": "https://bugzilla.suse.com/1222806"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-pydantic-2.6.4-2.1",
						"openSUSE Tumbleweed:python311-pydantic-2.6.4-2.1",
						"openSUSE Tumbleweed:python312-pydantic-2.6.4-2.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-pydantic-2.6.4-2.1",
						"openSUSE Tumbleweed:python311-pydantic-2.6.4-2.1",
						"openSUSE Tumbleweed:python312-pydantic-2.6.4-2.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-04-15T04:03:07Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2024-3772"
		}
	]
}
